R/zzz.R

```diff
@@ -1,5 +1,27 @@
 .lockbox_env <- new.env()
 
+.get_safely_from_envir <- function(target, default = NULL) {
+  res <- tryCatch(
+    get(target, envir = .lockbox_env)
+  , error = function(e) {default})
```

Comment:
This only errors if it doesn't exist, right? What about

```r
if (exists(target, envir = .lockbox_env, inherits = FALSE)) {
  get(target, envir = .lockbox_env)
} else {
  default
}
```

If that is the intended behavior, I suggest `.lockbox_env$session_id %||% default` instead of the function, which should be equivalent. Because of my mistake in defining `.lockbox_env`, your current approach could run into the snag where `target` is define in a parent environment: if there is a global variable `session_id` that will be used instead! Maybe you can fix zzz.R for me:

```r
.lockbox_env <- new.env(parent = emptyenv())
```